.idea/misc.xml

```diff
@@ -3,14 +3,29 @@
   <component name="DesignSurface">
     <option name="filePathToZoomLevelMap">
       <map>
-        <entry key="..\:/SDP/palFinder/app/src/main/res/layout-w936dp/activity_login.xml" value="0.258879781420765" />
```

Comment:
Not sure how this will function but the paths seem to be related to your local setup

Reply:
it's not my local setup but someone else, corrected it